COMMUNITY_MEMBERSHIP.md

```diff
@@ -133,7 +140,13 @@ The process for becoming a maintainer should be defined in the project or subpro
   - Identifying subtle or complex issues in designs and implementation PRs
 - Directly contributed to the subproject through implementation and / or review
 - Aligning with the overall project goals, specifications, and design principles. Bringing general questions and requests to the discussions as part of the specifications project.
-
+- [Open an issue](https://github.com/keptn/community/issues/new?template=membership.md&title=REQUEST%3A%20Maintainer%20membership%20for%20%3Cyour-GH-handle%3E)
```

Comment:
Can we try to set the title using: `title=REQUEST%3Membership%20maintainer%20for%3Cyour-GH-handle%3E`

Reply:
do you mean: `title=REQUEST%3AMembership%20maintainer%20for%3Cyour-GH-handle%3E` (please note the %3A instead of %3)

Reply:
any specific reason that the spaces are removed?
this is how it will look like 
`REQUEST:Membership maintainer for<your-GH-handle>`
I'd prefer a version with the spaces
`REQUEST: maintainer membership for <your-GH-handle>`

Reply:
Oh, I accidentally removed the `A` from `%3A` and the spaces. 
I prefer your suggestion too!